### ISSUE TITLE
Issue when upgrading to TW v2

### ISSUE THREAD

**Issue by RJFares** [tailwindlabs/tailwindcss-custom-forms#88]
After upgrading to Tailwind v2 (with postcss7 compat), requiring this plugin in my tailwind.config.js throws the following error on npm run:

```
Module build failed (from ./node_modules/css-loader/index.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/src/index.js):
TypeError: value.charCodeAt is not a function at
```
At this line in my scss: @tailwind components;

**Comment by web2033** [comment]
I think they recommend https://github.com/tailwindlabs/tailwindcss-forms for v2

**Comment by RJFares** [comment]
Probably time to mark this project as abandoned with a link in the description to the new tailwind forms

**Comment by deantheiceman** [comment]
How do we use the new one while keeping the same styles

**Comment by wobsoriano** [comment]
> I think they recommend https://github.com/tailwindlabs/tailwindcss-forms for v2

Looking at the commits, I think they're preparing for Tailwind 2 compatibility https://github.com/tailwindlabs/tailwindcss-custom-forms/pull/79

**Comment by themustafaomar** [comment]
F

**Comment by themustafaomar** [comment]
@web2033 

Unfortunately, https://github.com/tailwindlabs/tailwindcss-forms is just doing some resets to forms not an alternative to this repository 😭

**Comment by rhukster** [comment]
I switched from custom to `tailwindcss-forms` + some custom CSS to apply my form styling (which I previously had set in `tailwind.config.js`)  based on class names as I can't easily modify every field individually.

**Comment by wobsoriano** [comment]
Yep, looks like they're not planning on supporting Tailwind CSS 2

https://twitter.com/adamwathan/status/1330846547039367169

**Comment by web2033** [comment]
It's all in docs https://tailwindcss.com/docs/upgrading-to-v2#update-typography-and-forms-plugins

> If you are using `@tailwindcss/custom-forms`, you will want to migrate to `@tailwindcss/forms` which replaces it.
Learn more about the new forms plugin in the release notes.
> 
> The `@tailwindcss/custom-forms` plugin is not compatible with Tailwind CSS v2.0.

**Comment by EmmanuelJPaul** [comment]
> After upgrading to Tailwind v2 (with postcss7 compat), requiring this plugin in my tailwind.config.js throws the following error on npm run:
> 
> ```
> Module build failed (from ./node_modules/css-loader/index.js):
> ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/src/index.js):
> TypeError: value.charCodeAt is not a function at
> ```
> 
> At this line in my scss: @tailwind components;
> 
> Is this plugin ready for the tailwind upgrade yet or might it be another issue from my side?

I had the same issue. Remove the `require('@tailwindcss/ui')`  from plugins in tailwind.config.js and recompile.
